### ISSUE TITLE
Accessibility Audit: Mirador incompatible with JAWS 2019 (Needs Confirmation) 

### ISSUE THREAD

**Issue by brittnylapierre** [ProjectMirador/mirador#3879]
Description: An accessibility audit of Mirador was performed . During the test, a user using JAWS 2019 indicated they could not use this software with Mirador. 

Supporting Comments:
- DEAF-BLIND – Windows 10/JAWS 2019/Focus 14 braille/Chrome - Viewer 2 does not work at all with JAWS 2019 on Windows 10. JAWS 2020 used instead. 
- DEAF-BLIND – Windows 10/JAWS 2019/Focus 14 braille/Chrome - The page does not show any content except the download links.

**Comment by marlo-longley** [comment]
Priority High if confirmed. Action: run this scenario and confirm.
